src/resources/i18n/translations/en.json

```diff
@@ -712,9 +712,9 @@
   "Use custom web domain": "Custom web domain",
   "Use External Storage For Downloads": "To external storage card",
   "Username": "Username",
-  "value_tag_preview_boost": "A boost sends a donation.",
-  "value_tag_preview_boostagram": "A boostagram also adds a public message.",
-  "value_tag_preview_streaming": "Stream sends a donation for every minuted listened.",
+  "value_tag_preview_boost": "A <i>boost</i> sends a donation.",
+  "value_tag_preview_boostagram": "A <i>boostagram</i> also adds a public message.",
+  "value_tag_preview_streaming": "<i>Stream</i> sends a donation for every minuted listened.",
```

Comment:
It's fine if you want to italicize these, but this won't render correctly in the React Native Text component. Making this change would need it's own Github issue. I'll remove these after merging.